packages/suite/src/components/suite/SuiteLayout/index.tsx

```diff
@@ -167,9 +189,23 @@ const SuiteLayout = (props: SuiteLayoutProps) => {
     const { guideOpen } = useSelector(state => ({
         guideOpen: state.guide.open,
     }));
-    const { openGuide } = useActions({
+    const { openGuide, closeGuide } = useActions({
         openGuide: guideActions.open,
+        closeGuide: guideActions.close,
     });
+    useHotkeys(
+        'f1',
+        e => {
+            e.preventDefault();
+            if (guideOpen) {
+                closeGuide();
+            } else {
+                openGuide();
+            }
+        },
+        {},
+        [guideOpen],
```

Comment:
`closeGuide` and `openGuide` should be also mentioned as dependencies along with `guideOpen` to be safe, would be nice to add eslint warning (`react-hooks/exhaustive-deps`) for that

Reply:
this eslint warning should there (at least it works in other components)

Reply:
I see, this needs `additionalHooks` option, because it's a "custom" hook...

Reply:
maybe it would be nice to have also "esc" key to close (if guide is opened)?

Reply:
esc added, deps added. 

_(Rebased and squashed)_